### ISSUE TITLE
Define data-rule-target for keyboard-trap rules

### ISSUE THREAD

**Issue by jkodu** [auto-wcag/auto-wcag#237]
As a part of exporting test cases, it is essential to ear mark target elements in each test case, against which the rule needs to be tested.

Keyboard trap rules are missing this detail:
- https://auto-wcag.github.io/auto-wcag/rules/SC2-1-2-no-keyboard-trap-non-standard-navigation.html
- https://auto-wcag.github.io/auto-wcag/rules/SC2-1-2-no-keyboard-trap-standard-navigation.html

Ideally relevant test cases should have `data-rule-target` defined, like so:
`<audio data-rule-target src="../test-assets/moon-audio/moon-speech.mp3" controls></audio>
`

Refer rule https://auto-wcag.github.io/auto-wcag/rules/SC1-2-1-media-alternative-audio.html for usage example.

**Comment by jkodu** [comment]
RDF is going to solve for this. Hence closing.
